packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -247,9 +248,38 @@ export class MessageWidget extends Component {
       ) {
         // Send notification of new message
         const avatars = props.avatar.get('items');
+        let cardsLength = 0;
+        let message;
+
+        if (lastActivityFromThis.object) {
+          message = lastActivityFromThis.object.displayName;
+
+          if (lastActivityFromThis.object.cards && lastActivityFromThis.object.cards.length > 0) {
+            cardsLength = lastActivityFromThis.object.cards.length;
+            const sharedCardsNotificationMessage = formatMessage(messages.sharedCards, {cardsLength});
+
+            if (lastActivityFromThis.verb === API_ACTIVITY_VERB.POST) {
+              message = sharedCardsNotificationMessage;
+            }
+            else if (lastActivityFromThis.verb === API_ACTIVITY_VERB.SHARE && lastActivityFromThis.object.files
+              && lastActivityFromThis.object.files.items
+              && lastActivityFromThis.object.contentCategory === MESSAGE_CONTAINS_IMAGES) {
```

Comment:
A more descriptive constant name for this should be `ACTIVITY_OBJECT_CONTENT_CATEGORY_IMAGES`.

Reply:
changed the descriptive name